### ISSUE TITLE
missing nuget package SocketMobile.ScanAPI.Xamarin

### ISSUE THREAD

**Issue by jaybowman** [SocketMobile/singleentry-xamarin-ios#1]
I've downloaded the zip and I'm trying to get the application to compile. i've also donloaded the all the nuget packages from socketmobile site, socketmobile-scanapi-xamarin-ios-10-2-111-0.nupkg. i continue to get a nuget error that it can't find the SocketMobile.ScanIPI.Xamarin package. Also where is the CalaBash package?

Jay Bowman

**Comment by jaybowman** [comment]
I've made some progress. i have the calabash package. but when i try to add the socketmobile scan api for xamarin-ios I tet this error 

Unable to find metadata of SocketMobile.ScanAPI.Xamarin.iOS.10.2.111  
OK, i renamed the nuget package to .zip and unzipped the contents. Then i added the Scanapi.dll as a reference to the VS project. I will see if this works.

**Comment by jaybowman** [comment]
i did not mean to close the issue.

**Comment by LenOttSocketMobile** [comment]
Jay, trying to see what your issue is and get help from our Xamarin 'experts'!
We have no Calabash package...

**Comment by LenOttSocketMobile** [comment]
Jay, I think we need a bit more details? If you want you can email us directly at developers@socketmobile.com

**Comment by asfilo** [comment]
Were you able to resolve this issue? We're having the same issue trying to use SocketMobile_ScanAPI_Xamarin_iOS_10-3-7-36.nupkg.

**Comment by bitdozer** [comment]
I'm having a similar issue.  After downloading from the developer portal, cannot install the 10.3.7.36 nuget.  

The file exists in the local nuget path and is visible for installation, but install always fails.  Here is a package manager console session where the package and version are verified as available, yet install still fails:

<img width="833" alt="screenshot 2017-06-11 08 48 07" src="https://user-images.githubusercontent.com/7613458/27011521-cb5a1b96-4e82-11e7-9284-84d917e22abc.png">

Have tried this in the latest version of both Xamarin Studio and Visual Studio on Mac.

**Comment by asfilo** [comment]
We had the same issue when we were using a Nuget source that was just a folder on our system. We ended up having to "host" the package by creating and adding the package to a local feed

The key is to utilize the "nuget add" command. 

Here is a link to some documentation: https://docs.microsoft.com/en-us/nuget/hosting-packages/local-feeds

FYI, we utilized the built in package feed that is part of Visual Studio Team Services.
